oteps/XXXX-extending-attributes-to-support-complex-values.md

```diff
@@ -179,47 +162,58 @@ Semantic conventions will be updated with the following guidance
 
 ## Trade-offs and mitigations
 
-1. Allowing arbitrary objects as attributes is convenient but increases the risk of
-   including large, sensitive, mutable, non-serializable, or otherwise problematic
-   data in telemetry.
+### Backends don't support `AnyValue` attributes
 
-   It is RECOMMENDED to enforce `AnyValue` compatibility at the API level.
-   Users and instrumentations SHOULD define custom `AnyValue`-compatible models
-   to minimize misuse and reduce performance overhead.
+While it should be possible to record complex data in telemetry, many backends do not
+support it well - or at all - which can result in individual complex attributes
+or entire telemetry items being dropped or rejected.
 
-   OTel SDKs MAY provide convenience methods to convert arbitrary objects to `AnyValue`.
-   If such convenience is provided, it is RECOMMENDED to limit supported types
-   to primitives, arrays, standard library collections, named tuples, JSON objects, and
-   similar structures.
+In the future, support for complex attributes may be negotiated via OpAMP,
+allowing this behavior to be configured based on backend capabilities without
+requiring user intervention.
```

Comment:
if we're talking about the future, I wonder how useful this is, since it seems not a huge work for backends to make this choice and do their own json serialization

Reply:
I meant to leave a concrete suggestion 😄 
```suggestion
```